service/lib/dinstaller/manager.rb

```diff
@@ -135,6 +123,8 @@ def install
         proposal = ::Bootloader::ProposalClient.new.make_proposal({})
         logger.info "Bootloader proposal #{proposal.inspect}"
         @software.propose
+        # TODO: install required storage tools
```

Comment:
well, where it have to be? in live CD? if so, then we need different software backend as software used here for propose targets target system and not live one.

Reply:
I am not sure. This is something we need to discuss.